Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -36,7 +36,7 @@ namespace WebCore {
 
 static CSSCounterStyleDescriptors::Ranges translateRangeFromStyleProperties(const StyleProperties& properties)
 {
-    auto ranges = properties.getPropertyCSSValue(CSSPropertySystem).get();
+    auto ranges = properties.getPropertyCSSValue(CSSPropertyRange).get();
```

Comment:
Do we have test coverage for this mistake? I don’t see a change to a test result.

Reply:
The test failures can be seen with this patch: https://github.com/WebKit/WebKit/pull/10644

Reply:
RenderListMarker integration + https://github.com/WebKit/WebKit/pull/10644  are needed before we can see any kind of test coverage changes. Though I'm very confident WPT has some visual coverage for this.

Reply:
Same `get()` problem here. Pre-existing but wrong.